filebeat/tests/system/test_prospector.py

```diff
@@ -514,3 +514,45 @@ def test_close_older_file_rotation_and_removal(self):
                 max_timeout=10)
 
         filebeat.check_kill_and_wait()
+
+    def test_skip_symlinks(self):
+        """
+        Test that symlinks are skipped
+        """
+        self.render_config_template(
+            path=os.path.abspath(self.working_dir) + "/log/*",
+        )
+
+        os.mkdir(self.working_dir + "/log/")
+        testfile = self.working_dir + "/log/test-2016.log"
+        symlink_file = self.working_dir + "/log/test.log"
+
+        # write first line
+        with open(testfile, 'a') as file:
+            file.write("Hello world\n")
+
+        if os.name == "nt":
+            import win32file
+            win32file.CreateSymbolicLink(symlink_file, testfile, 1)
```

Comment:
The link is to a file so the last parameter should be 0. https://msdn.microsoft.com/en-us/library/windows/desktop/aa363866(v=vs.85).aspx

Hopefully that will fix the issue with 7z on AppVeyor.